helpers.js

```diff
@@ -69,6 +69,33 @@ export const verificationFail = async ({
   );
 };
 
+export function expectedMultibasePrefix(cryptosuite) {
+  const b64urlNoPadSuites = ['ecdsa-sd-2023', 'bbs-2023'];
+
+  if(b64urlNoPadSuites.includes(cryptosuite)) {
+    return {
+      prefix: 'u',
```

Comment:
We can add the rest of these in a separate PR: https://github.com/multiformats/multibase